From 599a80369d476eaacd29cb6a7613f37f712c2734 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Tue, 13 Aug 2013 21:30:18 +0200
Subject: [PATCH] Add additional search path to pkgconfig configuration

Some components include luna-service.h instead of luna-service2/luna-service.h so
it's usefull to have both include paths supported in our pkgconfig configuration.

Upstream-Status: pending (http://github.com/openwebos/luna-service2)

Open-webOS-DCO-1.0-Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 files/pkgconfig/luna-service2.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/pkgconfig/luna-service2.pc.in b/files/pkgconfig/luna-service2.pc.in
index 9ccf595..c72ed26 100644
--- a/files/pkgconfig/luna-service2.pc.in
+++ b/files/pkgconfig/luna-service2.pc.in
@@ -23,4 +23,4 @@ Name: @CMAKE_PROJECT_NAME@
 Description: @WEBOS_PROJECT_SUMMARY@
 Version: @WEBOS_COMPONENT_VERSION@
 Libs: -L${libdir} -l@CMAKE_PROJECT_NAME@
-Cflags: -I${includedir}
+Cflags: -I${includedir} -I${includedir}/luna-service2
-- 
1.8.1.2

